docker-compose-edinburgh-1.0.0.yml

```diff
@@ -127,18 +127,18 @@ services:
         - 'POSTGRES_USER=kong'
 
   kong-migrations:
-    image: "kong:0.13.0"
+    image: "kong:1.0.3"
```

Comment:
This image isn't arm compatible
```
ubuntu@ip-172-31-27-205:~$ uname -m
aarch64
ubuntu@ip-172-31-27-205:~$ docker run -it --rm kong:1.0.3 /bin/sh
Unable to find image 'kong:1.0.3' locally
1.0.3: Pulling from library/kong
docker: no matching manifest for unknown in the manifest list entries.
```

Reply:
Currently, the project doesn't use multi-arch manifests and publishes different image names for arm than for x86

Reply:
@anonymouse64 is this reply about EdgeX not using multi-arch or Kong?

Reply:
sorry EdgeX not using multi-arch, so the docker-compose file you're referencing is meant to be x86 specific and there will be a separate one for armhf or arm64

Reply:
See https://github.com/edgexfoundry/developer-scripts/issues/104 also

Reply:
Ah perfect I was concerned EdgeX was going to try to run ☝️ on non x86 as is which I predict sadness :)